### PR TITLE
Create Helm chart for deploying Keycloak

### PR DESCRIPTION
**Pull request by DanBondarenko**
Resolves https://github.com/keeleinstituut/tv-tolkevarav/issues/61

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,5 +14,327 @@ gradle-app.setting
 # Cache of project
 .gradletasknamecache
 
-# # Work around https://youtrack.jetbrains.com/issue/IDEA-116898
-# gradle/wrapper/gradle-wrapper.properties
+
```
Comment (thenouan) [inline .gitignore:25]: I don't know if it makes sense to include everything out of the box into gitignore. I would approach this as on-request, add entry when it's needed. There's no one gitignore that suits everything and is rather project specific. Current project does not rely on anything else than Dockerfile and Helm chart so I would just list common ignored files according to these two only.

As the code base increases then such ignores (with stuff that's not relevant) makes it more confusing - sometimes you can use gitignore to find out the results of some steps that are recreatable, builds, caches, etc, and gives you understanding that you need to do something to create these ignored files

File: helm/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: "0.1.0"  # Replaced during continuous delivery pipeline
```
Comment (thenouan) [inline helm/Chart.yaml:18]: Replaced using --version?

File: helm/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "" # Replaced during continuous delivery pipeline
```
Comment (thenouan) [inline helm/Chart.yaml:24]: Replaced using --app-version?

File: helm/templates/_helpers.tpl [hunk 4]
```diff
@@ -0,0 +1,62 @@
+{{/*
```
Comment (thenouan) [inline helm/templates/_helpers.tpl:1]: This file is created by helm boilerplate? Are there any manual changes in here as well?

Comment (DanBondarenko) [reply helm/templates/_helpers.tpl:1]: No manual changes, just boilerplate.

File: helm/Chart.yaml (hunk 3 above)
Comment (DanBondarenko) [reply helm/Chart.yaml:24]: Yes, exactly. 
image

File: helm/Chart.yaml (hunk 2 above)
Comment (DanBondarenko) [reply helm/Chart.yaml:18]: Yep.

image

File: .gitignore (hunk 1 above)
Comment (DanBondarenko) [reply .gitignore:25]: OK. I’ll remove.

Comment (DanBondarenko) [reply .gitignore:25]: @thenouan I kept only rules related to OS and IDE. Let me know if it looks OK to you now.

**Review by thenouan — APPROVED** [review]

**Pull request merged**